zipline/data/loader.py

```diff
@@ -295,9 +295,21 @@ def ensure_treasury_data(symbol, first_date, last_date, now, environ=None):
 def _load_cached_data(filename, first_date, last_date, now, resource_name,
                       environ=None):
     if resource_name == 'benchmark':
-        from_csv = pd.Series.from_csv
+        # We expect a series here.
+        def from_csv(path):
+            return pd.read_csv(
+                path,
+                parse_dates=[0],
+                index_col=0,
+                header=None,
+            ).tz_localize('UTC').iloc[:, 0]
```

Comment:
I think you just want to pass `squeeze=True` to `read_csv` instead of slicing here